src/transformers/models/beit/modeling_beit.py

```diff
@@ -1151,6 +1167,12 @@ def __init__(self, config: BeitConfig) -> None:
         self.beit = BeitModel(config, add_pooling_layer=False)
 
         # FPNs
+        if len(self.config.out_indices) != 4:
```

Comment:
This error is now going to be raised for everyone trying to create a semantic segmentation model from a default config

Reply:
Yes, see my comment above regarding whether we want this or not